packages/store-ui/src/atoms/Bullets/Bullets.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import type { Meta, Story } from '@storybook/react'
```

Comment:
Can you write your story in mdx like in here?
https://github.com/vtex/faststore/pull/782

Reply:
Sure!